wear/src/main/res/drawable/ic_lightbulb.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:fillColor="#FF000000"
```

Comment:
I'm seeing a black lightbulb on a grey background when choosing a complication because of this. Everywhere else the icon is being tinted, and I don't think Wear OS ever has a white/light background in native system menus, so maybe set this color to white to match other complications in the list?
![A list of complication options with the app's complication showing a black lightbulb on a grey background, barely visible](https://user-images.githubusercontent.com/8148535/171686235-f88422e0-3f98-420f-bd5b-b1cc10c91da0.png)

Reply:
Should definitely not be black indeed. What's interesting is that my samsung watch has a range of colored icons in this menu, so I'm thinking to go home assistant blue?